author.rb

```diff
@@ -0,0 +1,22 @@
+require 'uuid'
+require_relative 'items'
+
+class Author
+  attr_accessor :first_name, :last_name, :id, :items
```

Comment:
The `id` property is listed as private on the UML diagram, so I think it shouldn't have a getter or setter
![image](https://user-images.githubusercontent.com/101157253/214369525-96fa9cd3-45fd-4b89-8ff6-73f00bee578d.png)

Reply:
Oh, and the same goes for `items`

Reply:
@Dadadon , I agree with Felipe, if you fix that, you can merge